arangod/Aql/AstNode.cpp

```diff
@@ -2860,6 +2860,15 @@ void AstNode::setStringValue(char const* v, size_t length) {
   value.length = static_cast<uint32_t>(length);
 }
 
+void AstNode::setStringValue(std::string const& v) {
+  auto length = size_t{v.length()};
+  auto value = std::make_unique<char[]>(length);
+
+  std::memmove(value.get(), v.c_str(), length);
```

Comment:
```suggestion
  std::memcpy(value.get(), v.c_str(), length);
```

althigh I do not think it matters much performance wise:
1) memmove can be used on overlapping ranges.
2) memcpy cannot be used on overlapping ranges, and saves check for those (faster) here you clearly have non-overlapping ranges as you do not copy within the same string.